packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -213,7 +282,9 @@ const Autocomplete = <
       options={options}
       readOnly={isReadOnly}
       renderInput={renderInput}
-      value={value}
+      value={localValue}
+      inputValue={undefined}
+      isOptionEqualToValue={isOptionEqualToValue}
```

Comment:
Are we always setting `inputValue` to undefined for some reason?

Reply:
I was testing something out and didn't revert this back. Will fix.